WISE-Community/WISE
# Enabled differential loading and reduced bundle size

**Summary**
Size of javascript files that are loaded in the homepage reduced from 4.3Mb to 2.5Mb. Notable changes are:
- Added support for differential loading
- Moved most imports from polyfill.ts to individual apps that actually use them
- Removed hopscotch library and instead use imports
- Removed Moment-js locales from the bundle

**How to test**
Run ```npm install``` and test both dev and prod builds and make sure that everything (at, cm, student, preview) works as before. Test these in particular:
- Draw, label, graph components
- real-time (score, comment, pause/unpause)
- summernote (rubric/html component authoring, teacher/student notebooks)

Resolves #2324
